walk-in-interview/src/main/webapp/log-in/applicant-log-in/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../../style.css">
+  </head>
+
+  <body>
+    <div class="log-in-section">
+      <ul>
+        <li id="account">
+          <label for="account-input"></label>
+          <input id="account-input" name="account">
+        </li>
+  
+        <li id="password">
+          <label for="password-input"></label>
+          <input id="password-input" name="password">
+        </li>
+      </ul>
+
+      <button id="otp"></button>
```

Comment:
I thought we decided on the docs to just do otp and no password for the applicant?

Reply:
it is only otp~ maybe i change the text to make it clear